grpc-server-security-jwt/build.gradle

```diff
@@ -0,0 +1,20 @@
+
+dependencies {
+
+    annotationProcessor "io.micronaut:micronaut-inject-java:$micronautVersion"
+
+    api project(":grpc-server-runtime")
+    api "io.micronaut:micronaut-inject:$micronautVersion"
+    api "io.micronaut:micronaut-runtime:$micronautVersion"
+    api "com.nimbusds:nimbus-jose-jwt:9.4.2"
+
+    implementation("io.micronaut.security:micronaut-security-jwt:$micronautSecurityVersion") {
+        exclude group: 'io.micronaut', module: 'micronaut-http'
```

Comment:
Decided to exclude the `http` dependencies since they are not used and might not be necessary on the classpath of consumers.